### ISSUE TITLE
How to set testID on Checkbox

### ISSUE THREAD

**Issue by rollsroyc3** [react-native-training/react-native-elements#1964]
**Explain what you did**
I cannot assign the testID prop to the Checkbox  component.

**Expected behaviour**
Checkbox to use the testID prop I give it so I can run tests and interact with checkbox.

**Describe the bug**
Get error when running test because it cant find testID.

**Possible Solution**
Assign testID prop to Component.

Im also curious if this an issue with just the Checkbox, or if there are other elements that dont pass testID prop for automated e2e testing.

**Comment by rollsroyc3** [comment]
Looking at the source, I see a testID for the text, but not for the component which contains the onPress handler.

https://github.com/react-native-training/react-native-elements/blob/master/src/checkbox/CheckBox.js

**Comment by iRoachie** [comment]
Added in #1965. Will be available in 1.2.0

**Comment by rollsroyc3** [comment]
Thanks @iRoachie for the quick PR. Im curious though shouldn't this be a standard across all the elements? I saw in another [PR](https://github.com/react-native-training/react-native-elements/issues/1848) that you worked on that allows a custom testID. Not that I need that functionality, just as people use more and more elements they would expect feature parity across them.

**Comment by iRoachie** [comment]
Sound like a good idea. If you can make an issue with a list, we can work through each one

**Comment by rollsroyc3** [comment]
Will do, Im curious what is the `attributes` prop for? Ive seen that throughout the code...

**Comment by iRoachie** [comment]
It’s the object rest https://v8.dev/features/object-rest-spread. It’s all the other props that aren’t restructured

**Comment by rollsroyc3** [comment]
Thanks, wasn't aware you could do that to get the other props.

**Comment by Bene8493** [comment]
I'm using 1.2.7 and testID is still not available for CheckBox

**Comment by Nonag** [comment]
Also on 1.2.7 and the CheckBox does not provide a testID property :<

**Comment by Nonag** [comment]
I just realised that with #1965 the testID for the RNE CheckBox is always "checkbox" and you can not pass an individual value for the testID property like I was expecting.

So if you want to interact with it in your tests, e.g. using Detox, you have to do something like this to target it:

```javascript
element(by.id('checkbox').withAncestor(by.id('WrapperThatSupportsTestIDProperty')))
```

**Comment by iRoachie** [comment]
@Nonag It's fixed in https://github.com/react-native-elements/react-native-elements/pull/2272. But not released yet.

**Comment by stereodenis** [comment]
Still no testID for Checkbox component

**Comment by dooleyb1** [comment]
> @Nonag It's fixed in #2272. But not released yet.

Was this ever released?
